### ISSUE TITLE
Date Picker Localization

### ISSUE THREAD

**Issue by Farid-Najafloo** [moxi9/phpfox#617]
the date picker localization script can't load because of folders address was changed on version 4.x
Recommendation:
codes that need to change:
file:
include/library/phpfox/template/template.class.php

1-Line 1139:
Old:
```php
$sLocalDatepicker = PHPFOX_STATIC .'jscript/jquery/locale/jquery.ui.datepicker-' . strtolower(Phpfox_Locale::instance()->getLangId()) . '.js';
```
New:
```php
$sLocalDatepicker = PHPFOX_DIR.PHPFOX_STATIC .'jscript/jquery/locale/jquery.ui.datepicker-' . strtolower(Phpfox_Locale::instance()->getLangId()) . '.js';
```
2-Line 1143:
Old:
```php
$sFile = str_replace(PHPFOX_STATIC . 'jscript/', '', $sLocalDatepicker);
```
 New:
```php
$sFile = str_replace(PHPFOX_DIR.PHPFOX_STATIC . 'jscript/', '', $sLocalDatepicker)
```

**Comment by phpfoxdev** [comment]
Hi, in what page the script could not load?

**Comment by Farid-Najafloo** [comment]
Creating an event

**Comment by phpfoxdev** [comment]
The DatePicker lib still loads fine as our end. Kindly provide more information. Also, do you mean this issue: https://github.com/moxi9/phpfox/issues/623? if so, it's a layout issue.

**Comment by Farid-Najafloo** [comment]
my problem is'n about layout!

this  a simple debugging:
the "ui.js" is an official file

```php

                // include/library/phpfox/template/template.class.php : Line 1139
				$sLocalDatepicker = PHPFOX_STATIC .'jscript/jquery/locale/jquery.ui.datepicker-' . strtolower(Phpfox_Locale::instance()->getLangId()) . '.js';
				
                //me
                $fileAddress1 = PHPFOX_STATIC .'jscript/jquery/ui.js';
                $fileAddress2 = PHPFOX_DIR.PHPFOX_STATIC .'jscript/jquery/ui.js'; // My Recommendation
                
                echo("file_exists($fileAddress1)?  ". (file_exists($fileAddress1)?"YES":"NO").'<br />'); // = NO
                echo("file_exists($fileAddress2)?  ". (file_exists($fileAddress2)?"YES":"NO").'<br />'); // = YES
                exit();
                //me
                
				if (file_exists($sLocalDatepicker)) // this block will not run never!
				{
					$sFile = str_replace(PHPFOX_DIR.PHPFOX_STATIC . 'jscript/', '', $sLocalDatepicker);
					$this->setHeader(array($sFile => 'static_script'));					
				}
				

 ```

**Comment by phpfoxdev** [comment]
Fixed, will be released in next version.

**Comment by Data66PF** [comment]
Fixed in v4.0.10 *staff note YPFOXNEUTRON-79 *
